src/azure-cli-core/azure/cli/core/application.py

```diff
@@ -31,13 +31,11 @@ def __init__(self, argv):
 
     def get_command_table(self):
         import azure.cli.core.commands as commands
-        # Find the first noun on the command line and only load commands from that
```

Comment:
Wasn't this an optimization to speed up loading times? It seems removing it would slow down a bunch of cases.